operator/pkg/central/values/translation/translation.go

```diff
@@ -173,6 +177,34 @@ func getCentralComponentValues(c *platform.CentralComponentSpec) *translation.Va
 		}
 		cv.AddChild("exposure", &exposure)
 	}
+
+	if c.CentralDBEnabled() {
+		cv.AddChild("db", getCentralDBComponentValues(c.DB))
+	}
+
+	return &cv
+}
+
+func getCentralDBComponentValues(c *platform.CentralDBSpec) *translation.ValuesBuilder {
+	cv := translation.NewValuesBuilder()
+	cv.SetBoolValue("enabled", true)
+	cv.SetBoolValue("external", c.IsExternal())
+
+	if c.IsExternal() && c.ConnectionStringOverride != nil {
+		cv.SetError(errors.New("if external is set to true, connection string override must also be set"))
```

Comment:
I think we should similarly error if no password secret name is specified in `IsExternal` case? Otherwise the only options are:
- passwordless account
- password embedded in the connection string

Right?

Neither of these sounds very secure :thinking: